### PR TITLE
Comments/Reader: remove the totalCommentsCount reducer

### PR DESCRIPTION
**Pull request by samouri**
*WIP*

some cleanup from https://github.com/Automattic/wp-calypso/pull/15882

### CODE REVIEW

File: client/state/comments/selectors.js [hunk 1]
```diff
@@ -109,6 +101,34 @@ export const getPostCommentsTree = createSelector(
...
+
+/**
+ * Get total number of loaded comments for a post
+ *
+ * @param {Object} state redux state
+ * @param {Number} siteId site identification
+ * @param {Number} postId site identification
+ * @return {Number} total comments count on the server. if not found, assume infinity
+ */
+export const getPostLoadedComments = ( state, { siteId, postId, status } ) => {
```
Comment (samouri) [inline client/state/comments/selectors.js]: should rename to `getLoadedCommentsCount`

**Comment by apeatling** [comment]
@samouri Is this one still WIP? If not let's close it up.

**Comment by apeatling** [comment]
@samouri Will close, but please reopen when you make updates.

**Comment by samouri** [comment]
thanks for closing this. its definitely dead. some small parts of janitorial work may be worth reviving in the future.

**Pull request closed without merging**